prost-build/src/config.rs

```diff
@@ -835,37 +835,31 @@ impl Config {
         Ok(())
     }
 
-    /// Compile `.proto` files into Rust files during a Cargo build with additional code generator
-    /// configuration options.
-    ///
-    /// This method is like the `prost_build::compile_protos` function, with the added ability to
-    /// specify non-default code generation options. See that function for more information about
-    /// the arguments and generated outputs.
-    ///
-    /// The `protos` and `includes` arguments are ignored if `skip_protoc_run` is specified.
+    /// Loads `.proto` files as a `FileDescriptorSet`. This allows inspection of the descriptors
+    /// before calling `prost_build::compile_fds`. A possible use case it to compute `Config`
+    /// attributes by introspecting what is actually present in the `.proto` files before
+    /// generating the code.
```

Comment:
I suggest rewording this a bit. The square brackets cause `cargo doc` to create a link to the type.

```suggestion
    /// Loads `.proto` files as a [`FileDescriptorSet`]. This allows inspection of the descriptors
    /// before calling [`Config::compile_fds`]. This could be used to change [`Config`] 
    /// attributes after introspecting what is actually present in the `.proto` files.
```